### ISSUE TITLE
Postgresql support

### ISSUE THREAD

**Issue by alondahari** [pbogut/vim-dadbod-ssh#7]
Hi, not sure if I have a configuration issue or work needs to be done to support postgresql.

Running the command:
`:DB ssh://my-ssh-profile:postgres://username:pa$$word@rds_cluster_url:5432/db_name`

(removed actual values from the above string)

My ssh config:
```
Host my-ssh-profile
  User ec2-user
  HostName port_of_bastion_host
  Port 22
```

Getting the following error:
<img width="1331" alt="CleanShot 2023-10-16 at 23 01 13@2x" src="https://github.com/pbogut/vim-dadbod-ssh/assets/5546798/ea2fc7f3-c8a2-431c-911a-c0bb5dda9e85">

Thanks!

**Comment by pbogut** [comment]
tl;dr: Try `postgresql://` instad of `postgres://` , and make sure ssh connection works.

I am using it with Postgres, so it should work fine. And your URI looks correct.

First of all, it seems like the tunnel was not created and it timed out. Either the configuration is wrong or the timeout is too low for your server. Make sure this works fine in the terminal first by running `ssh my-ssh-profile`. Maybe there was some interaction needed on the first connection.

I was trying to reproduce your error and it looks like I have the same error if I run `DB ...` from a file that is not `.sql`. It works fine from `.sql` files, but I get the error from others. I will have to investigate to check where the problem is exactly. As a workaround, you can use the `postgresql://` schema, which seems to work fine.

**Comment by alondahari** [comment]
Alright, using postgresql seems to work. Thanks!
